src/pages/MovieListPage/MovieListPage.tsx

```diff
@@ -46,42 +38,40 @@ const MovieListPage: React.FC = () => {
   useEffect(() => {
     if (responseMovies) {
       setMovieList(responseMovies.data);
-      setSelectedMovie(null);
     }
-  }, [responseMovies, setSelectedMovie]);
+  }, [responseMovies]);
 
-  const handleAddMovie = (moviePayload: IMovieInfo): void => {
-    console.log("New movie added:", moviePayload);
-    setIsAddMovieModalOpened(false);
+  const updateSearchParams = (params: Record<string, string>) => {
+    navigate({
+      pathname: RoutePaths.Home,
+      search: searchParams.toString(),
+    });
+    setSearchParams((currentParams) => {
+      const [key, value] = Object.entries(params)[0];
+      const newParams = new URLSearchParams(currentParams);
+      newParams.set(key, value);
```

Comment:
![image](https://github.com/user-attachments/assets/cce79eab-ea3d-4a1c-b255-3e5bb71e7a40)

Can we delete it?
![image](https://github.com/user-attachments/assets/39198cb2-8dcb-432f-9a38-f42109c0d214)

Reply:
Fixed